docs/developer/sre-doc.md

```diff
@@ -0,0 +1,327 @@
+        prometheus-server-1aa1aaaa1a1a1111`
+*   Dashboard is on https://localhost:11111/
+    *   As the server has a self signed cert, please use 'thisisunsafe' to
+        continue past the error message
+
+##### Prometheus configuration GCP
+
+```
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+  external_labels:
+      environment: turbinia-gcp_node
+
+scrape_configs:
+  - job_name: 'turbinia-gcp'
+    gce_sd_configs:
+        # The GCP Project
+        - project: {PROJECT_NAME}
+          zone: {ZONE}
+          filter: labels.turbinia-prometheus=true
+          refresh_interval: 120s
+          port: 8000
+```
+
+The Turbinia hosts that need to be scrapped have the label 'turbinia-prometheus'
+set by the Terraform scripts and are auto discovered by Prometheus.
+
+#### Grafana
+
+*   Grafana pulls metrics from Google monitoring (OS metrics) and application
+    metrics from Prometheus.
+*   Grafana displays dashboard for both the OS and Turbinia application metrics.
+*   Datasource 1: Google Monitoring of gcp project
+*   Datasource 2: `http://prometheus-server-1aa1aaaa1a1a1111:9090/`
+*   Listens on port 3000
+*   Current the username is **"admin"** and password is generated by terraform when it 
+deploys Grafana.
+
+**WARNING**: You are responsible for securly storing the Grafan admin password after terraform
+deployment is complete.
+
+*   SSH access
+    *   `ssh -i ~/.ssh/google_compute_engine -L 11111:localhost:3000
+        Grafana-server-1aa1aaaa1a1a1111`
+*   Dashboard is on https://localhost:11111/
+    *   As the server has a self signed cert, please use 'thisisunsafe' to
+        continue past the error message
+
+
+
+### Importing a new dashboard to Grafana
+
+*   Login to [Grafana](#grafana)
+*   Click the “+” sign on the left sidebar and then select “import”.
+*   Then copy/paste the json file from the dashboard you want to import and
+    click “Load”.
+
+### Exporting a dashboard from Grafana
+
+*   Login to [Grafana](#grafana)
+*   From the dashboard, select the “dashboard Setting” on the upper right corner
+*   Click on “JSON Model” and copy the contents of the textbox.
+*   In order to import this to another dashboard, follow [these steps](#importing-a-new-dashboard-to-grafana).
+
+### How to update prometheus config
+
+To update the prometheus config file or add new rules to prometheus:
+
+*   SSH to the host VM as mentioned [here](#prometheus) .
+*   To change the config:
+    *   Change and save the contents of `/etc/prometheus/prometheus.yml`
+    *   Restart the prometheus container by running `docker restart
+        container_id`
+        *   To find the container\_id run`docker ps`
+*   To add new prometheus rules:
+    *   Create or update an existing rule file. [Prometheus docs]
+        (https://prometheus.io/docs/prometheus/latest/configuration/recording_rules)
```

Comment:
Doesn't look like this is rendering the link correctly.  Maybe remove the whitespace between the link title and link target?